cantabular-import/dp-cantabular-csv-exporter.yml

```diff
@@ -23,4 +23,5 @@ services:
             RECIPE_API_URL:         "http://dp-recipe-api:22300"
             CANTABULAR_URL:         "http://dp-cantabular-server:8491"
             CANTABULAR_EXT_API_URL: "http://dp-cantabular-api-ext:8492"
+            LOCAL_OBJECT_STORE:     "http://minio:9000"
```

Comment:
Should the 9000 be 9001   ?
(just asking as it was changed elsewhere for other reasons)

Reply:
Nope, it needs to be 9000 to the other services on the network but 9001 when accessing from outside

Reply:
OK.
Thanks.